ts/utils/messageIdToString.ts

```diff
@@ -0,0 +1,3 @@
+export function messageIdToString (msgId: Uint8Array): string {
+  return (new Uint8Array(msgId)).toString()
```

Comment:
Use https://github.com/achingbrain/uint8arrays#tostringarray-encoding--utf8

Reply:
:+1: I guess we can use base16 or base64? The msg id won't be guaranteed to be valid utf8

Reply:
Is it not safe to use `.toString()` directly? While the resulting string is not good for readability it may be good enough for comparison

Reply:
@wemeetagain we can use `base64` 👍 
@dapplion if we need to revert it back to Uint8Array anytime in the future, it will cause issues.

Reply:
I wish that library had type definitions :sweat_smile: